.env-dist

```diff
@@ -0,0 +1,7 @@
+DJANGO_DEBUG=True
+ALLOWED_HOSTS=localhost, 127.0.0.1
+DJANGO_SECRET_KEY=some secret
+DJANGO_HMAC_KEY=some secret
+DATABASE_URL=mysql://root:@localhost:3306/oneanddone
```

Comment:
Minor but maybe you could add a password here, like `mysql://root:password@localhost:3306/oneanddone` to better indicate to the users where they can input their mysql credential, if any. Also this may need some explaining in the README file.